app/src/main/java/com/duckduckgo/app/httpsupgrade/db/HttpsBloomFilterSpecDao.kt

```diff
@@ -20,20 +20,16 @@ import android.arch.persistence.room.Dao
 import android.arch.persistence.room.Insert
 import android.arch.persistence.room.OnConflictStrategy
 import android.arch.persistence.room.Query
+import com.duckduckgo.app.httpsupgrade.model.HttpsBloomFilterSpec
+import javax.inject.Singleton
 
 @Dao
-interface HttpsUpgradeDomainDao {
+@Singleton
+interface HttpsBloomFilterSpecDao {
 
     @Insert(onConflict = OnConflictStrategy.REPLACE)
-    fun insertAll(domains: List<HttpsUpgradeDomain>)
-
-    @Query("select count(1) > 0 from https_upgrade_domain where domain = :domain")
-    fun hasDomain(domain: String) : Boolean
-
-    @Query("delete from https_upgrade_domain")
-    fun deleteAll()
-
-    @Query("select count(1) from https_upgrade_domain")
-    fun count(): Int
+    fun insert(specification: HttpsBloomFilterSpec)
 
+    @Query("select * from https_bloom_filter_spec limit 1")
```

Comment:
Uppercase the SQL keywords?

Reply:
This isn't a pattern we currently use in our codebase (nearly all the daos are lowercase). How do you feel about discussing the convention in our next fdri meeting and then updating the code universally according to the decision we make?

Reply:
You're right; we use a mishmash of upper and lower case, even in the same query 😞

Cool, ignore it for this PR